### PR TITLE
add miningtycoon

### PR DESCRIPTION
**Pull request by nextypefinance**
add miningtycoon

##### Twitter Link:
https://twitter.com/NEXTYPE1

##### List of audit links if any:
https://www.certik.org/projects/nextype

##### Website Link:
https://nextype.finance/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://nextype.finance/img/logo1024.png

##### Current TVL:
$1.34M

##### Chain:
heco

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
nextype-finance

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
10742

##### Description/bio (to be shown on DefiLlama):
NEXTYPE is an integrated application ecosystem among game, NFT and DeFi that can be cross-chain (cross platform), which will create a more efficient and free NFT asset swapping protocol for the blockchain world, and provide users with high-quality multiform Dapps based on games + DeFi, games + NFT synthesis, and NFT + DeFi.

##### Token address and ticker if any:
NT 
https://hecoinfo.com/token/0x8b70512b5248e7c1f0f6996e2fde2e952708c4c9

##### Category (Yield/Dexes/Lending/Minting):
GameFi / Yield / Minting

##### [Optional] ETH addresss (to receive a LlamaPunk):
0x0A924500c30B7d7556DC969Bc894a4Cb0CDfaf91

### CODE REVIEW

File: projects/miningtycoon/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const tokenFarm = '0xBdfbeecF52bCfF5aa8cc1B8A4B737B2Af3D1BA2F';
+const NTToken = '0x8b70512b5248e7c1f0f6996e2fde2e952708c4c9';
+const USDTToken = '0xa71edc38d189767582c38a3145b5873052c3e47a';
+
+
+async function tvl() {
+	let balances = {};
+	
+	const NTBalance = (await sdk.api.abi.call({
```
Comment (0xngmi) [inline projects/miningtycoon/index.js:12]: This seems to be staking, it'd be better to move it to the staking section

File: projects/miningtycoon/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	
+	balances = {};
+	balances['heco:'+NTToken] = NTBalance;
+	balances['heco:'+USDTToken] = USDTBalance;
+		
+	return balances;
+}
+
+module.exports = {
+	ethereum:{
```
Comment (0xngmi) [inline projects/miningtycoon/index.js:37]: Given that this is all on heco I think this should be classified as heco TVL instead of ethereum one

**Review by 0xngmi — COMMENTED** [review]
Left a few minor comments, will merge and apply the changes locally. Let me know if you disagree with anything and open to discuss it :)

**Comment by realdealshaman** [comment]
Hey @nextypefinance, thanks for contributing to Defillama. Your LlamaPunk has been sent: https://opensea.io/0x0a924500c30b7d7556dc969bc894a4cb0cdfaf91

**Pull request merged**